### ISSUE TITLE
No module named sklearn

### ISSUE THREAD

**Issue by chrisgorgo** [BIDS-Apps/ndmg#11]
While building the image
```
Installing collected packages: cython, coverage, requests, docopt, coveralls, wget, nibabel, nilearn, dipy, scikit-learn, sklearn, networkx, PyYAML, colorama, docutils, futures, python-dateutil, jmespath, botocore, s3transfer, pyasn1, rsa, awscli, boto3, plotly, pyvtk, cycler, matplotlib
  Running setup.py install for docopt: started
    Running setup.py install for docopt: finished with status 'done'
  Running setup.py install for wget: started
    Running setup.py install for wget: finished with status 'done'
  Running setup.py install for nibabel: started
    Running setup.py install for nibabel: finished with status 'done'
  Running setup.py install for nilearn: started
    Running setup.py install for nilearn: finished with status 'error'
    Complete output from command /usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-gpqnOY/nilearn/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-CPwHf7-record/install-record.txt --single-version-externally-managed --compile:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-gpqnOY/nilearn/setup.py", line 52, in <module>
        module_check_fn(is_nilearn_installing=True)
      File "<string>", line 110, in _check_module_dependencies
      File "<string>", line 60, in _import_module_with_version_check
    ImportError: ('No module named sklearn', 'Module "sklearn" could not be found. See http://nilearn.github.io/introduction.html#installation for installation information.')
```

**Comment by gkiar** [comment]
Weird, building locally I get:
```
Gregs-MacBook-Pro [master] $ docker build -t bids/ndmg .
Sending build context to Docker daemon 30.72 kB
Step 1/14 : FROM bids/base_fsl:5.0.9-3
 ---> 8ae4dfad2a59
........
Step 14/14 : ENTRYPOINT ndmg_bids
 ---> Running in 9b889e800b32
 ---> 33ee8bc991f6
Removing intermediate container 9b889e800b32
Successfully built 33ee8bc991f6
```

... I'll restart the build and then investigate further if that fails again.

**Comment by gkiar** [comment]
fixed in 8f0c88d
